src/graph/test/DataTest.cpp

```diff
@@ -269,6 +289,62 @@ TEST_F(DataTest, InsertVertex) {
         };
         ASSERT_TRUE(verifyResult(resp, expected));
     }
+    // Test same prop name diff type in diff tags
+    {
+        cpp2::ExecutionResponse resp;
+        std::string cmd = "INSERT VERTEX person(name, age),employee(name) "
+                          "VALUES hash(\"Joy\"):(\"Joy\", 18, 123),"
+                          "hash(\"Petter\"):(\"Petter\", 19, 456)";
+        auto code = client_->execute(cmd, resp);
+        ASSERT_EQ(cpp2::ErrorCode::SUCCEEDED, code);
+    }
+    {
+        cpp2::ExecutionResponse resp;
+        std::string cmd = "INSERT EDGE schoolmate(likeness) VALUES "
+                          "hash(\"Joy\")->hash(\"Petter\"):(90)";
+        auto code = client_->execute(cmd, resp);
+        ASSERT_EQ(cpp2::ErrorCode::SUCCEEDED, code);
+    }
+    {
+        cpp2::ExecutionResponse resp;
+        std::string cmd = "GO FROM hash(\"Joy\") OVER schoolmate YIELD $^.person.name,"
+                          "schoolmate.likeness, $$.person.name, $$.person.age,$$.employee.name";
+        auto code = client_->execute(cmd, resp);
+        ASSERT_EQ(cpp2::ErrorCode::SUCCEEDED, code);
+        std::vector<std::tuple<std::string, int64_t, std::string, int64_t, int64_t>> expected = {
+            {"Joy", 90, "Petter", 19, 456},
+        };
+        ASSERT_TRUE(verifyResult(resp, expected));
+    }
+    // Test same prop name same type in diff tags
+    {
+        cpp2::ExecutionResponse resp;
+        std::string cmd = "INSERT VERTEX person(name, age),interest(name) "
```

Comment:
for this test case:

insert vertex person(name, age), interest(name)  

/* 1. can I miss any field? e.g. person(name), interest(name) 
if no, please add a case*/

"VALUES hash(\"Bob\"):(\"Bob\", 19, \"basketball\")";

/* 2. can I miss any filed? e.g."VALUES hash(\"Bob\"):(\"Bob\", /*19,*/ \"basketball\")";

if no, please add a case */